### PR TITLE
Prefix sort code cleanup

### PR DESCRIPTION
**Pull request by xiaoxmeng**
Differential Revision: D65721084

### CODE REVIEW

**Review by tanjialiang — APPROVED** [review]

File: velox/exec/PrefixSort.cpp [hunk 1]
```diff
@@ -24,56 +24,61 @@ namespace {
...
 // If the alignment is changed from 8 to 4, you need to change bitswap64
 // to bitswap32.
-const int32_t kAlignment = 8;
+static constexpr int32_t kAlignment = 8;
 
 template <typename T>
 FOLLY_ALWAYS_INLINE void encodeRowColumn(
     const PrefixSortLayout& prefixSortLayout,
-    const uint32_t index,
+    uint32_t index,
```
Comment (jinchengchenghh) [inline velox/exec/PrefixSort.cpp:32]: Please use column_index_t for the column index

File: velox/exec/PrefixSort.cpp [hunk 2]
```diff
@@ -267,39 +279,43 @@ void PrefixSort::sortInternal(
...
+            return compareAllNormalizedKeys(lhs, rhs);
+          });
     }
   }
-  // 4. Output sorted row addresses.
-  for (int i = 0; i < rows.size(); i++) {
-    rows[i] = getAddressFromPrefix(prefixes + i * entrySize);
+
+  // Output sorted row addresses.
+  for (int i = 0; i < rows.size(); ++i) {
```
Comment (jinchengchenghh) [inline velox/exec/PrefixSort.cpp:317]: for (auto i = 0, the rows.size may exceed int

File: velox/exec/PrefixSort.h [hunk 3]
```diff
@@ -47,22 +47,22 @@ struct PrefixSortLayout {
...
-  /// It equals to 'numNormalizedKeys == 0', a little faster.
-  const bool noNormalizedKeys;
+  /// It equals to 'numNormalizedKeys != 0', a little faster.
+  const bool hasNormalizedKeys;
 
   /// Whether the sort keys contains non-normalized key.
   const bool hasNonNormalizedKey;
 
-  /// Offsets of normalized keys, used to find write locations when
+  /// Offsets of normalized keys, used to find write locations whenbu
```
Comment (jinchengchenghh) [inline velox/exec/PrefixSort.h:56]: typo whenbu

**Review by jinchengchenghh — COMMENTED** [review]
Some minors, thanks!

File: velox/exec/PrefixSort.h [hunk 4]
```diff
@@ -16,7 +16,7 @@
 #pragma once
 
 #include "velox/common/base/PrefixSortConfig.h"
-#include "velox/common/memory/MemoryAllocator.h"
+
```
Comment (jinchengchenghh) [inline velox/exec/PrefixSort.h:19]: Drop the blank line.

**Review by jinchengchenghh — APPROVED** [review]

**Pull request closed without merging**